packages/neuron-ui/src/utils/is.ts

```diff
@@ -9,3 +9,13 @@ export const isMainnet = (networks: Readonly<State.Network[]>, networkID: string
 export const isSuccessResponse = (res: Pick<ControllerResponse, 'status'>): res is SuccessFromController => {
   return res.status === ResponseCode.SUCCESS
 }
+
+export const isReadyByVersion = (targetVersion: number, lastVersion: number | null) => {
+  if (lastVersion === null || Number.isNaN(+lastVersion)) {
```

Comment:
is `+lastVersion` always a number?

Reply:
Yes, it consists of a major version and a minor version.

Reply:
If so, I am concerned about the `Number.isNaN(+lastVersion)` is always `true`, which makes this `if` check meaningless.

Reply:
Sure, it's not necessary, I'll remove it.

Reply:
Updated in https://github.com/nervosnetwork/neuron/pull/1743/commits/2e9d8493772231751a4ac1c2e810865374036335